frontend/src/Admin/components/NavBar.js

```diff
@@ -49,6 +49,7 @@ export default class NavBar extends Component {
                 <NavLink tag={Link} to='/' onClick={this.props.switchToJudge}>
                   View as Judge |
                 </NavLink>
+                {/* TODO: Move the | out of the link */}
```

Comment:
Does this TODO need to be removed?

Reply:
Nope, at least not in this PR – just making a note here because Rob wanted to change this.